app/editor/src/features/admin/sources/services/ScheduleAdvanced.tsx

```diff
@@ -88,7 +92,7 @@ export const ScheduleAdvanced: React.FC<IScheduleAdvancedProps> = (props) => {
             </Button>
           )}
           {index !== undefined && (
-            <Button onClick={handleDone} variant={ButtonVariant.secondary}>
+            <Button type="submit" onClick={handleDone} variant={ButtonVariant.secondary}>
```

Comment:
Does `onClick` get fired if user presses enter in the form somewhere else?
It's very odd to have the button as the `submit` but it's using the `secondary` variant.  Why is this?

Reply:
oops this was meant to be removed!